### DISCUSSION TITLE
Lantern Backpack Upgrade

### DISCUSSION THREAD

**Discussion by Conga0** [P3pp3rF1y/SophisticatedBackpacks#898] [Ideas]
You know how there's mods which let you hold a torch, glowstone or a lava bucket in your hand and provide an active light source for your immediate surroundings?

It'd be awesome to see an upgrade like that for the backpack, some kind of attachable lantern like you can see in some media.
A light revealing what's around you would be perfect for mining expeditions and late night trips outside.

**Comment by I2pRandom** [comment]
Would require a optional dependency of a DynamicLights mod, as many mods already fill this purpose. But also there are already so many DynamicLights mods to pick from, along with mods that nest in X/Y/Z mod for Rubidium/Embeddium

**Comment by Conga0** [comment]
The main thing I want from this is a dynamic light I don't need to actively be holding an item to achieve.

Holding torches, glowstones and lanterns is cool but it's annoying to place them down when I need to inevitably right click, that's why I want it to be mounted on the backpack
